dotCMS/src/main/java/com/dotmarketing/util/StringUtils.java

```diff
@@ -316,7 +318,25 @@ public static String sanitizeFileName(final String fileName) {
       return FileUtil.sanitizeFileName(fileName);
       
     }
-    
+
+    private static final Pattern singleQuotedLiteral = Pattern.compile("(?:^|\\s)*'([a-zA-Z0-9-_]+)'(?:$|\\s)*", Pattern.MULTILINE);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field singleQuotedLiteral has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5002120)